src/main/resources/config/config.yaml

```diff
@@ -64,5 +64,5 @@ jerseyClientConfiguration:
 baseUrl: ${BASE_URL}
 publicApiUrl: ${PUBLICAPI_URL}
 productsUiPayUrl: ${PRODUCTSUI_PAY_URL}
-
+productsUiConfirmUrl: ${PRODUCTSUI_CONFIRMATION_URL}
```

Comment:
suggestion: Normally I would put all the local urls as default values for all these URL configs if environment variable is not present. So that we don't have to keep maintaining `pay-scripts`. 
For e.g. 
```
productsUiPayUrl: ${PRODUCTSUI_PAY_URL:-https://productsui.pymnt.localdomain/pay}
```

Reply:
This can be a separate PR as none of the previous env vars were set this way and thus it is a 'feature' request.

Reply:
its fine. Not the end of the world!

Reply:
I agree with your comment and I shall do it in the next PR for all the vars.